src/clean_cnpj_info_dataset.py

```diff
@@ -0,0 +1,78 @@
+from itertools import chain
+import json
+import numpy as np
+import pandas as pd
+
+def decompose_main_activity(value):
+    struct = json.loads(value.replace('\'', '"'))
+    return pd.Series([*struct]). \
+        rename_axis({'code': 'main_activity_code', 'text': 'main_activity'})
+
+def decompose_secondary_activities(value):
+    struct = json.loads(value.replace('\'', '"'))
+    if bool(struct) and struct[0]['text'] != 'Não informada':
```

Comment:
Is there a specific reason to use `bool(struct)` instead of just `struct`? We could also use `Dict.get(key)` to (eventually) avoid `KeyError`:

```python
if struct and struct[0].get('text') != 'Não informada':
    …
```